lib/webauthn/authenticator_response.rb

```diff
@@ -25,7 +25,7 @@ def initialize(client_data_json:, relying_party: WebAuthn.configuration.relying_
     end
 
     def verify(expected_challenge, expected_origin = nil, user_verification: nil, rp_id: nil)
-      expected_origin ||= relying_party.origin || raise("Unspecified expected origin")
+      expected_origin ||= relying_party.origin || relying_party.allowed_origins || raise("Unspecified expected origin")
```

Comment:
What do you think about adding a deprecation warning for the users of the gem using the `origin` attribute? That might help migrating to the using only the `allowed_origins` attribute in the future.